instrument-starters/opentracing-spring-cloud-jdbc-starter/pom.xml

```diff
@@ -47,6 +47,12 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>common</artifactId>
```

Comment:
what does this adds?

Reply:
all objects in latest opentracing jdbc are created as proxy. this dep adds util class to create proxy.